cafebabel/templates/articles/macros.html

```diff
@@ -8,6 +8,14 @@
   {% endif %}
 {% endmacro %}
 
+{% macro profile_image(author) %}
+  {% if author.profile.has_image %}
+    <div style="background-image:url({{ author.profile.image_url }})" class=image></div>
```

Comment:
If I remember well, you do this to be able to resize into a centered squared with preserved ratio.
As this is hackish, could you comment in the code _why_ you need to put the image in background instead of `<img src>`?

Reply:
yep